src/pudl/transform/vcerare.py

```diff
@@ -365,6 +365,8 @@ def check_rows(context: AssetCheckExecutionContext) -> AssetCheckResult:
     logger.info("Check VCE RARE hourly table is the expected length")
 
     # Define row counts for fast/full etl
+    # TODO 2024-12-27: make this check row counts per year instead of having
```

Comment:
Agree, but we'll probably eventually wind up with distribution tests that do want fast vs. full specified and can't be parameterized by year, so the solution will probably come in handy anyways?